listenbrainz/config.py.sample

```diff
@@ -120,6 +120,7 @@ LASTFM_PROXY_URL = 'http://localhost:8080/'
 SERVER_ROOT_URL = 'http://localhost'
 WEBSOCKETS_SERVER_URL = 'http://localhost:8081'
 MUSICBRAINZ_OAUTH_URL = 'https://musicbrainz.org/oauth2/userinfo'
+LISTENBRAINZ_LABS_API_URL = 'https://labs.api.listenbrainz.org'
```

Comment:
Could you add this to `consul_config.py.ctmpl`? Exactly this line should be ok, we don't need to make it variable.